resources/dicts/patrols/general/border.json

```
@@ -2360,8 +2360,27 @@
 	    ],
 	    "stat_trait": ["bloodthirsty", "arrogant", "vengeful", "fierce", "cold", "grumpy"],
 	    "weight": 30,
-            "art": "gen_bord_newcatkittypet1"
+            "art": "gen_kittypet_friends_with_dog1"
 
+        },
+        {
+            "text": "p_l lashes out to swipe at the dog but is interrupted by a kittypet rushing to stand in {PRONOUN/p_l/poss} way. p_l's fur bristles for a moment, but the kittypet tells {PRONOUN/p_l/object} to calm down, that this is {PRONOUN/n_c:0/poss} friend. As p_l recovers from.. whatever this is, {PRONOUN/p_l/subject} offers the kittypet a place to stay in c_n- this kittypet's skill in communicating with dogs may come in handy later.",
+            "exp": 10,
+	    "new_cat": [
+		["status:warrior", "backstory:kittypet1"]
```

```suggestion
		["status:warrior", "backstory:kittypet1", "kittypet", "exists]
```
up to you if you accept this suggestion! but adding these would allow for existing kittypets to be acquired via this event.